### PR TITLE
Bump gems for 6.0.1

### PR DESCRIPTION
**Pull request by andrewvc**
Bump gems for 6.0.1 feature freeze

### CODE REVIEW

File: Gemfile.jruby-2.3.lock.release [hunk 1]
```diff
@@ -131,48 +131,48 @@ GEM
     json-schema (2.6.2)
       addressable (~> 2.3.8)
     kramdown (1.14.0)
-    logstash-codec-cef (5.0.1-java)
+    logstash-codec-cef (5.0.2-java)
```
Comment (andrewvc) [inline Gemfile.jruby-2.3.lock.release]: Just an update of the gemspec summary

File: Gemfile.jruby-2.3.lock.release [hunk 2]
```diff
@@ -131,48 +131,48 @@ GEM
...
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-codec-collectd (3.0.7)
+    logstash-codec-collectd (3.0.8)
```
Comment (andrewvc) [inline Gemfile.jruby-2.3.lock.release]: Gemspec summary

File: Gemfile.jruby-2.3.lock.release [hunk 3]
```diff
@@ -131,48 +131,48 @@ GEM
...
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-codec-dots (3.0.5)
+    logstash-codec-dots (3.0.6)
```
Comment (andrewvc) [inline Gemfile.jruby-2.3.lock.release]: Gemspec summary

File: Gemfile.jruby-2.3.lock.release [hunk 4]
```diff
@@ -131,48 +131,48 @@ GEM
...
+    logstash-codec-msgpack (3.0.7-java)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       msgpack (~> 1.1)
-    logstash-codec-multiline (3.0.7)
+    logstash-codec-multiline (3.0.8)
       jls-grok (~> 0.11.1)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       logstash-patterns-core
-    logstash-codec-netflow (3.7.0)
+    logstash-codec-netflow (3.8.3)
```
Comment (andrewvc) [inline Gemfile.jruby-2.3.lock.release]: Fixed a race condition https://github.com/logstash-plugins/logstash-codec-netflow/blob/master/CHANGELOG.md

File: Gemfile.jruby-2.3.lock.release [hunk 5]
```diff
@@ -186,96 +186,96 @@ GEM
       rspec (~> 3.0)
       rspec-wait
       stud (>= 0.0.20)
-    logstash-filter-aggregate (2.7.0)
+    logstash-filter-aggregate (2.7.2)
```
Comment (andrewvc) [inline Gemfile.jruby-2.3.lock.release]: bugfix: fix synchronisation issue at Logstash shutdown (#75)

File: Gemfile.jruby-2.3.lock.release [hunk 6]
```diff
@@ -186,96 +186,96 @@ GEM
...
-    logstash-filter-metrics (4.0.4)
+    logstash-filter-metrics (4.0.5)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       metriks
       thread_safe
-    logstash-filter-mutate (3.1.6)
+    logstash-filter-mutate (3.1.7)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-ruby (3.0.4)
+    logstash-filter-ruby (3.1.1)
```
Comment (andrewvc) [inline Gemfile.jruby-2.3.lock.release]: File based ruby script support.

File: Gemfile.jruby-2.3.lock.release [hunk 7]
```diff
@@ -420,13 +420,13 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       mail (~> 2.6.3)
       mime-types (< 3)
-    logstash-output-file (4.1.1)
+    logstash-output-file (4.2.1)
```
Comment (andrewvc) [inline Gemfile.jruby-2.3.lock.release]: New write_behavior feature. Value can be "append" (default) or "overwrite". If "append", events will be appended to the end of the file. If "overwrite", the file will only contain the last event written.

**Comment by andrewvc** [comment]
There are a lot of bumps due to a mass update of gemspecs, I added comments for the exceptions

File: Gemfile.jruby-2.3.lock.release (hunk 6 above)
Comment (jsvd) [reply Gemfile.jruby-2.3.lock.release]: I suggest not including this in a .patch release

File: Gemfile.jruby-2.3.lock.release (hunk 7 above)
Comment (jsvd) [reply Gemfile.jruby-2.3.lock.release]: I see no reason to use this new minor on a .patch release

File: Gemfile.jruby-2.3.lock.release [hunk 8]
```diff
@@ -186,96 +186,96 @@ GEM
...
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-fingerprint (3.1.1)
+    logstash-filter-fingerprint (3.1.2)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       murmurhash3
-    logstash-filter-geoip (5.0.1-java)
+    logstash-filter-geoip (5.0.2-java)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-grok (3.4.3)
+    logstash-filter-grok (3.4.5)
```
Comment (IrlJidel) [inline Gemfile.jruby-2.3.lock.release]: Does this need to be 4.0.0 now as 3.4.5 has been yanked

Comment (jsvd) [reply Gemfile.jruby-2.3.lock.release]: indeed this should be removed

**Comment by jsvd** [comment]
after a few releases with bugfixes on them, this .lock file needs the following updates:

logstash-filter-ruby -> 3.1.2 : contains a bugfix in threadsafety for jruby
logstash-filter-grok -> 4.0.0 : contains a fix for a performance regression with the timer execution

**Pull request closed without merging**